qpylib/abstract_qpylib.py

```diff
@@ -171,6 +161,9 @@ def to_json_dict(self, python_obj, classkey=None):
         @return dict object containing key:value pairs for the python
         objects fields. Useable with JSON REST.
         """
+        print(type(python_obj))
```

Comment:
Why is the object being printed?

Reply:
Good spot. It was there to debug a python 3 issue. I'll remove it.